test/CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 aux_source_directory(${CMAKE_CURRENT_SOURCE_DIR} TEST_SOURCES)
+if (APPLE)
+	# don't want to test mknod - requires super-user privileges
+	add_definitions(-DFBR_TEST_NO_MKNOD)
+else()
+	# OSX doesn't use librt, while linux boxes do
+	set(FBR_TEST_RTLIB rt)
+endif(APPLE)
```

Comment:
Specifying `APPLE` in `endif()`, while omitting it in `else()` seems inconsistent. libevfibers CMake sources specify the condition in all control statements, although newer CMakes relaxed this to simple `()`.